photospp.sh

```diff
@@ -0,0 +1,41 @@
+package: PHOTOSPP
+version: "%(tag_basename)s-ship%(defaults_upper)s"
+source: https://github.com/PMunkes/PHOTOSPP
+tag: v3.61
+requires:
+  - HepMC
+  - ROOT
+  - pythia
+  - Tauolapp
+---
+#!/bin/sh
+
+export  HEPMCLOCATION="$HEPMC_ROOT"
+
+rsync -a $SOURCEDIR/* .
+
+./configure --enable-debug --with-hepmc=$HEPMC_ROOT --with-pythia8=$PYTHIA_ROOT --with-tauola=$TAUOLAPP_ROOT --prefix=$INSTALLROOT CFLAGS="$CFLAGS" CXXFLAGS="$CFLAGS"
```

Comment:
I suppose you did try running `$SOURCEDIR/configure`, right? Also, please use `rsync -a $SOURCEDIR/  .` (no `*`).